spinta/datasets/components.py

```diff
@@ -94,6 +94,7 @@ def __init__(self):
 class ExternalBackend(Backend):
     engine: Engine = None
     schema: sa.MetaData = None
+    accept_types: set = None
```

Comment:
Maybe set this to an empty set in `__init__`, in order to avoid dealing with `None`? It is always good idea, to avoid `None` if possible and in this case there is no need for `None`.